src/Notifications.svelte

```diff
@@ -168,51 +171,37 @@
   }
 
   export let timeout = 3000
+  export let closeable = true
 
 	let count = 0
-	let toasts = [ ]
+	let toasts = []
   let unsubscribe
 
-	function animateOut(node, { delay = 0, duration = 1000 }) {
-		function vhTOpx (value) {
-			var w = window,
-				d = document,
-				e = d.documentElement,
-				g = d.getElementsByTagName('body')[0],
-				x = w.innerWidth || e.clientWidth || g.clientWidth,
-				y = w.innerHeight|| e.clientHeight|| g.clientHeight;
-
-			return (y*value)/100;
-		}
-		
-		return {
-			delay,
-			duration,
-			css: t => `opacity: ${(t-.7) * 1}; transform-origin: top right;`
-		}
-	}
+	const animateOut = (node, { delay = 0, duration = 1000 }) => ({
+		delay,
+		duration,
+		css: t => `opacity: ${(t-.7) * 1}; transform-origin: top right;`
+	})
 
-	function createToast (msg, theme, to) {
+	const createToast = (msg, theme, to) => {
 		const background = themes[theme] || themes['default']
 		toasts = [{
 			id: count,
 			msg, 
 			background, 
 			timeout: to || timeout,
 			width: '100%'
-		}, ...toasts];
-		count = count + 1
+		}, ...toasts]
+		count += 1
   }
   
   unsubscribe = notification.subscribe(value => {
-    if (!value) { return }
+    if (!value) return
```

Comment:
I'm not a fan of braceless code, even for single lines, however, I'll add linting to the project to save bike-shedding :)